src/assets/plugins/filemanager/filemanager.js

```diff
@@ -50,6 +51,33 @@ if (!RedactorPlugins) var RedactorPlugins = {};
 					}, this)
 				});
 
+				var $tabBox3 = $('<div id="redactor-file-manager-delete-box" style="overflow: auto; height: 300px;" class="redactor-tab redactor-tab3">').hide();
```

Comment:
Image & File Manager plugins don't have a delete option from the box, so we should update it manually or hope that Imperavi will add this option later from the box.